lib/reader.js

```diff
@@ -1,32 +1,29 @@
 'use strict';
 
 const stream = require('readable-stream');
-const isUndefined = require('lodash.isundefined');
 const JSONStream = require('JSONStream');
 const pack = require('browser-pack');
 const mergeStream = require('merge-stream');
-const depsSort = require('deps-sort');
-
+const through = require('through2');
 
 /**
  *
  * We need to manipulate order in all bundles to make sure that the
- * order is an increased value accross all bundles.
+ * order is an increased value across all bundles.
  *
  */
 
 class Order extends stream.Transform {
-    constructor () {
+    constructor (order) {
         super({
             objectMode: true,
         });
-        this.order = 0;
+        this.order = order;
     }
 
     _transform (obj, encoding, next) {
-        if (!isUndefined(obj.order)) {
+        if (obj.entry === true) {
             obj.order = this.order;
-            this.order += 1;
         }
         this.push(obj);
         next();
```

Comment:
Pirk: We could change this to `next(null, obj)` and save a line. The code is equivalent.